src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/SqlTestsBase.cs

```diff
@@ -101,21 +101,13 @@ protected Management.Sql.SqlManagementClient GetSqlClient(RestTestFramework.Mock
             Management.Sql.SqlManagementClient client =
                 context.GetServiceClient<Management.Sql.SqlManagementClient>(
                     RestTestFramework.TestEnvironmentFactory.GetTestEnvironment());
-            if (HttpMockServer.Mode == HttpRecorderMode.Playback)
```

Comment:
why don't we need this anymore?

Reply:
We never needed it, it's set in the base class.  I think having it set redundantly here adds confusion for anyone looking at it.